.github/workflows/prbuild.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
         version: v1.55.2
         # TODO: re-enable linting tools package once https://github.com/projectcontour/contour/issues/5077
         # is resolved
-        args: --build-tags=e2e,conformance,gcp,oidc,none
+        args: --build-tags=e2e,conformance,gcp,oidc,none --out-${NO_FUTURE}format=colored-line-number
```

Comment:
I cannot figure out the explanation for `${NO_FUTURE}` but it seems that _many_ projects have it ([link](https://www.google.com/search?q=%22no_future%22+site%3Agithub.com)). It has no value during action execution so it evaluates to `--out-format`. If it _would_ have value during action execution, it would result in invalid argument name?  

It seems to me like the variable is added there only so that people get puzzled and can ask this question :rofl: 

I'd prefer we remove the variable and simply set `--out-format=colored-line-number` unless you know what it is about?

Reply:
https://github.com/golangci/golangci-lint-action/issues/485
I don't know either as this folks mentiond. Let me remove it